docs/en_US/Tutorial/WebUI.md

```diff
@@ -42,8 +42,8 @@ Click the tab "Intermediate Result" to see the lines graph.
 
 ![](../../img/webui-img/trials_intermeidate.png)
 
-The graph has a filter function. You can open the filter button. And then enter your focus point
-in the scape input. Simultaneously, intermediate result inputs can limit the intermediate's range.
+We set a filter function for the intermediate result because that your trials may have many intermediate results.
+You can use it if you want to see the trend of some trial. Import your focus intermediate result's step in the step input and then import the metric's min and max value you focus. Final, you can filter some trials to see the trend.
 
```

Comment:
Do you mean "input" other than "import" ?

Reply:
Maybe input is better.